docs/en/concepts-and-designs/lal.md

```diff
@@ -69,7 +64,6 @@ See examples below.
 ```groovy
 filter {
     json {
-        abortOnFailure true
```

Comment:
I think you still should add these docs, but committed and say, this is default behavior.